src/components/AddTodoForm.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import InputWithLabel from "./InputWithLabel";
+import propTypes from "prop-types";
+
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = React.useState('');
+
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+
+    }
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        onAddTodo({ title: todoTitle, id: Date.now() });
+
+        setTodoTitle('');
+    };
+    AddTodoForm.propTypes={
+        onAddTodo:propTypes.string
```

Comment:
The `onAddTodo` prop is a function, so the value here should be `propTypes.func`